.github/workflows/pull_request.yml

```diff
@@ -5,9 +5,28 @@ on: pull_request
 jobs:
   build:
     name: Test pull requests
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
think this is missing `macos-latest`?

Reply:
Tests won't pass for some reason, it doesn't find files in the sandboxed directories it seems. So I left it off for now until someone can debug.

Reply:
weird - in that case should remove the macos-specific bits farther down then probably

Reply:
Removed, and sure.